app/controllers/alaveteli_pro/base_controller.rb

```diff
@@ -37,6 +37,21 @@ def pro_user_authenticated?(reason_params = nil)
     return false
   end
 
+  # A pro-specific version of authenticated? that pushes the user to the pro
+  # signup route instead of the standard one
+  def pro_authenticated?(reason_params)
+    unless session[:user_id]
+      reason_params.merge!(pro: true)
```

Comment:
We discussed this in Slack. The gist was that actually, we don't need the custom route, so this can be cleaner:

```ruby
def pro_authenticated?(reason_params)
  authenticated?(reason_params.merge(pro: true))
end
```